index.html

```diff
@@ -4669,105 +4714,130 @@ <h3>Data First Approaches</h3>
     <h2>Internationalization Considerations</h2>
 
     <p>
-There are a number of internationalization considerations implementers should be
-aware of when publishing data described in this specification. As with any web
-standards or protocols implementation, ignoring internationalization makes it
-difficult for data to be produced and consumed across a disparate set of
-languages and societies, which would limit the applicability of the
-specification and significantly diminish its value as a standard.
+There are a number of internationalization considerations implementers
+are advised to be aware of when publishing data described in this specification.
+As with any web standards or protocols implementation, ignoring
```

Comment:
`As with any web standards or protocols implementation, ignoring `
-> `As with implementation of any web standard or protocol, ignoring `

Reply:
Fixed in f0b4f5063bad9633a8d0c5407fac3a2408ea4121.